test/WebSites/MvcTagHelpersWebSite/Views/MvcTagHelper_Home/Environment.cshtml

```diff
@@ -0,0 +1,46 @@
+﻿@addtaghelper "Microsoft.AspNet.Mvc.TagHelpers"
+
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <title>Environment</title>
+</head>
+<body>
+
+    <h2>Environment Tag Helper Test</h2>
+
+    <ul>
+        <environment names="Development">
+            <li>This will show</li>
+        </environment>
+        <environment names="development">
+            <li>This will show</li>
+        </environment>
+        <environment names="Development,  Production">
+            <li>This will show</li>
+        </environment>
+        <environment names="Production, Development">
+            <li>This will show</li>
+        </environment>
+        <environment names="Local,Development, Production">
+            <li>This will show</li>
+        </environment>
+        <environment names="Development,">
+            <li>This will show</li>
+        </environment>
+        <environment>
+            <li>This will show</li>
+        </environment>
+        <environment names="">
+            <li>This will show</li>
+        </environment>
+        <environment names=" ">
+            <li>This will show</li>
+        </environment>
+        <environment names=", ">
+            <li>This will show</li>
+        </environment>
```

Comment:
You told me in an earlier comment not to include the negative cases so I removed them :confounded: